python/search_completer/search_result_delegate.py

```diff
@@ -22,59 +21,71 @@
 from .utils import CompleterPixmaps
 
 
-class SearchResultDelegate(views.EditSelectedWidgetDelegate):
+class SearchResultDelegate(views.WidgetDelegate):
```

Comment:
Fixes some rendering issues with the completer.

Sometimes the first result would be gray rectangle, which turned out to because the widget was created in editor mode and cause some weird Qt-issues. Having the widget be a regular widget delegate fixed all of this. Becase of this however we now need to listen for selection events.